recipes/wg21-linear_algebra/all/conanfile.py

```diff
@@ -30,40 +30,37 @@ def _minimum_cpp_standard(self):
     def _minimum_compilers_version(self):
         return {
             "Visual Studio": "16",
-            "gcc": "8",
-            "clang": "8",
+            "msvc": "192",
+            "gcc": "10",
+            "clang": "12", # Should be 11 but https://github.com/conan-io/conan-docker-tools/issues/251
             "apple-clang": "11"
         }
 
     def validate(self):
+        compiler = self.settings.compiler
         if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, self._minimum_cpp_standard)
-        min_version = self._minimum_compilers_version.get(
-            str(self.settings.compiler))
+            check_min_cppstd(self, self._minimum_cpp_standard)
+        min_version = self._minimum_compilers_version.get(str(compiler))
         if not min_version:
-            self.output.warn("{} recipe lacks information about the {} "
-                             "compiler support.".format(
-                                 self.name, self.settings.compiler))
+            self.output.warn(f"{self.name} recipe lacks information about the "
```

Comment:
`self.output.warn` is not v2 compatible

Reply:
Now addressed here: https://github.com/conan-io/conan-center-index/pull/17456